manifests/02-deployment.yaml

```diff
@@ -31,6 +31,8 @@ spec:
           command:
           - cluster-ingress-operator
           env:
+            - name: RELEASE_VERSION
```

Comment:
If this isn't mapped somehow in `image-references` how will the value ever change?

Reply:
As you pointed me to the relevant conversation, good to know that this is the job of the cluster-autoscaler-operator now (https://github.com/openshift/cluster-autoscaler-operator/pull/67/files#diff-ff2a1a1bdac4c0edc4696c1d4bf9d684R29)